.github/workflows/ci.yml

```diff
@@ -103,14 +106,16 @@ jobs:
     - uses: actions/checkout@v1
     - name: install ninja
       run: sudo apt-get install ninja-build
+    - name: install lit and FileCheck
+      run: pip3 install lit FileCheck
```

Comment:
If we decide to go with this new test framework, can we make this a seperate PR.  Just add lit + FileCheck and perhaps convert one or two trivial tests?

Reply:
Yes, I agree 👍 see #3367 for what that would look like.